vmdb/app/models/container_replication_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ContainerReplicationController < ActiveRecord::Base
```

Comment:
@abonas @zeari @Fryguy yes there is a parallel in Amazon, they're called (just to create a little bit of confusion for us `services`):

http://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_CreateService.html

> **CreateService**
> Runs and maintains a desired number of tasks from a specified task definition. If the number of tasks running in a service drops below desiredCount, Amazon ECS will spawn another instantiation of the task in the specified cluster.

I would have had other comments for this patch. I guess we'll get to them later.